### PR TITLE
Development

### PR DESCRIPTION
**Pull request by pavank-v**
the code is still in development

### CODE REVIEW

File: assignment_5.1_pavan.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    else:
+        print('not found')
+        
+    price_of_book_fk = flipkart_book_soup.find('div',class_ = '_30jeq3 _16Jk6d')
+    print(price_of_book_fk.text)
+    '''end of flipkart'''
+    
+    '''amazon'''
+    print(amazon_url)
+    try:
```
Comment (pavank1301) [inline assignment_5.1_pavan.py:64]: here is the problem i couldn't able to get the url

File: assignment_5.1_pavan.py
Comment (praveen-pb) [inline assignment_5.1_pavan.py:1]: file name should not have multiple dots.

File: assignment_5.1_pavan.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+                        
+                else:
+                    return 'The Book is not Available in GoodReads'
+            else:
+                return 'The URL of the Book is not found in GoodReads'
+        else:
+            return 'No Response from GoodReads'
+                
+'''Dictionary which contains all the Books information'''
+book_list = {'book_1' : {'author_name' : 'Joseph Murphy', 'book_name' : 'The Power of Your Subconcious Mind'},
```
Comment (praveen-pb) [inline assignment_5.1_pavan.py]: Put these things inside a function

**Review by praveen-pb — APPROVED** [review]
This looks good, you can merge this. if possible try out cricbuzz. it will be little complicated, but worth learning.

File: assignment_5.1_pavan.py
Comment (pavank-v) [reply assignment_5.1_pavan.py:1]: changed the name

**Comment by pavank-v** [comment]
I have made all the changes like you suggested

**Pull request merged**